src/script/view_model/panel/TimedMessagesPanel.tsx

```diff
@@ -78,10 +77,9 @@ const TimedMessagesPanel: React.FC<TimedMessagesPanelProps> = ({onClose, onGoBac
   }, [globalMessageTimer]);
 
   const timedMessageChange = (value: number): void => {
-    if (activeConversation) {
-      const finalTimer = value === 0 ? null : value;
-      activeConversation.globalMessageTimer(finalTimer);
-      repositories.conversation.updateConversationMessageTimer(activeConversation, finalTimer);
+    if (activeConversation && value !== 0) {
```

Comment:
```suggestion
    if (activeConversation && value) {
```

Can be simplified :)

Reply:
let's keep it specific here for clarity